src/instrumentation_measurement/README.md

```diff
@@ -28,3 +28,110 @@ sudo docker run --rm --privileged --security-opt seccomp:unconfined \
 ```
 
 For other EVMs use respective `Dockerfile`s and use the `--evm` flag on the `measure` command, e.g. `measure --evm openethereum`
+
+
+# Test environment setup
+## Go Etherum Benchmark
+Compile benchmark program
+```
+cd geth_benchmark\tests\imapp_benchmark
+go build
+```
+## Erigon
```

Comment:
As previously noted: erigon is interesting, but we don't have the capacity to include it now, let's focus on geth benchmark and nethermind. I don't have the capacity to review or run erigon measurements. Pls move erigon changes to a new PR

Reply:
Erigon removed, still available in the branch: https://github.com/imapp-pl/gas-cost-estimator/tree/erigon_benchmark